test/erlfmt_pragma_SUITE.erl

```diff
@@ -0,0 +1,124 @@
+%% Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Insert pragma tests will also go into this file.

Reply:
I'd avoid creating a separate suite for this. In general performance of CT scales really poorly with a lot of small suites (they can't run in parallel). I think making a group in `erlfmt_SUITE` for those would be enough

Reply:
Hmmm, that is interesting.
I am worried that erlfmt_SUITE is already over 900 lines long.
It is starting to become hard to read and find things.

Reply:
I agree. I think it would make sense to extract the parser tests from `erlfmt_SUITE` into a separate file and leave `erlfmt_SUITE` for the more integration-style tests. This can be done in a new PR

Reply:
Oh okay, yes that can work :)